spec/rules_factory_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rspec"
+require "./rules_factory.rb"
+require "./item.rb"
+
+describe RulesFactory do
+  before(:each) do
+    @rule_factory = RulesFactory.new
```

Comment:
is it a rule_factory or a rules_factory? mismatch between class and variable.
I'd be okay with just naming the instance factory. Also, consider using let instead of before to instantiate the instance under test.